arcgis-ios-sdk-samples/Analysis/Viewshed (geoprocessing)/ViewshedGeoprocessing.storyboard

```diff
@@ -19,26 +20,27 @@
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="YLS-ms-81h" customClass="AGSMapView">
                                 <rect key="frame" x="0.0" y="44" width="414" height="852"/>
                             </view>
-                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="kPT-2F-sjn">
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="kPT-2F-sjn" userLabel="Instructions View">
                                 <rect key="frame" x="0.0" y="44" width="414" height="46"/>
                                 <subviews>
-                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Tap on the map to see all areas that are visible within 15 kilometres radius" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="qgh-ab-wUm">
+                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Tap on the map to see all areas that are visible within 15 kilometres radius" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="qgh-ab-wUm" userLabel="Instructions label">
                                         <rect key="frame" x="8" y="5" width="398" height="36"/>
                                         <fontDescription key="fontDescription" type="system" pointSize="15"/>
                                         <nil key="textColor"/>
                                         <nil key="highlightedColor"/>
                                     </label>
                                 </subviews>
-                                <color key="backgroundColor" red="0.97254901959999995" green="0.97254901959999995" blue="0.97254901959999995" alpha="0.9186059397163121" colorSpace="calibratedRGB"/>
+                                <color key="backgroundColor" systemColor="tertiarySystemBackgroundColor"/>
```

Comment:
I set the instructions view background color to `tertiarySystemBackgroundColor` because this view overlaps the map view; unlike other samples where the top of the map view is constrained to the bottom of the instructions view. I can update this sample so that the views do not overlap or update the others so that they do overlap. This is open for discussion as we should establish general rules for instruction views.

Reply:
In this case, please set the background view's background color to `statusLabelBackgroundColor` at top in the named colors section.

Also it is preferred to overlap the status label atop a map view or scene view. But it is up to you to make this change as it is trivial.